server/alertmanager_test.go

```diff
@@ -0,0 +1,71 @@
+package server
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/percona/pmm/api/alertmanager/amclient"
+	"github.com/percona/pmm/api/alertmanager/amclient/alert"
+	serverClient "github.com/percona/pmm/api/serverpb/json/client"
+	"github.com/percona/pmm/api/serverpb/json/client/server"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	pmmapitests "github.com/Percona-Lab/pmm-api-tests"
+)
+
+func TestAlertManager(t *testing.T) {
+	res, err := serverClient.Default.Server.GetSettings(nil)
+	require.NoError(t, err)
+	assert.True(t, res.Payload.Settings.TelemetryEnabled)
+	err = os.Setenv("PERCONA_TEST_ALERTMANAGER_RESEND_INTERVAL", "10s")
```

Comment:
Why do we set this env variable?

Reply:
This test allways works outside of docker container.

Reply:
> Why do we set this env variable?

i wanted a lower interval since the default is 30s

>  This test allways works outside of docker container.

You are right I didn't realize this, will remove it